lua/wikis/commons/Widget/Match/Countdown.lua

```diff
@@ -0,0 +1,58 @@
+---
+-- @Liquipedia
+-- page=Module:Widget/Match/Countdown
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Class = Lua.import('Module:Class')
+local Countdown = Lua.import('Module:Countdown')
+local DateExt = Lua.import('Module:Date/Ext')
+local Timezone = Lua.import('Module:Timezone')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+
+local UTC = Timezone.getTimezoneString{timezone = 'UTC'}
+
+---@class MatchCountdownProps
+---@field match MatchGroupUtilMatch
+
+---@class MatchCountdown: Widget
+---@operator call(MatchCountdownProps): MatchCountdown
+---@field props MatchCountdownProps
+local MatchCountdown = Class.new(Widget)
+
+---@return Widget?
+function MatchCountdown:render()
+	local match = self.props.match
+	if not match then
+		return nil
+	end
+
+	if match.timestamp == DateExt.defaultTimestamp then
+		return nil
+	end
+
+	local dateString
+	if match.dateIsExact then
+		local timestamp = DateExt.readTimestamp(match.date) + (Timezone.getOffset{timezone = match.extradata.timezoneid} or 0)
```

Comment:
why not use `match.extradata.timestamp`?

Reply:
Will be fixed in #6175